typedoc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "entryPoints": [
+    "./projects/ngx-piwik-pro/src/public-api.ts", 
+    "node_modules/@piwikpro/tracking-base-library/dist/interfaces/index.d.ts", 
```

Comment:
Question: shouldn't it be `node_modules/@piwikpro/react-piwik-pro/dist/index.d.ts` [like in nextjs](https://github.com/PiwikPRO/next-piwik-pro/blob/master/typedoc.json#L3) instead of `node_modules/@piwikpro/tracking-base-library/dist/interfaces/index.d.ts`